doc/api/cli.md

```diff
@@ -154,53 +208,80 @@ Note: v8 options allow words to be separated by both dashes (`-`) or underscores
 For example, `--stack-trace-limit` is equivalent to `--stack_trace_limit`.
 
 ### `--tls-cipher-list=list`
+<!-- YAML
+added: v4.0.0
```

Comment:
If it helps, here's the commit where that was added: https://github.com/nodejs/node/commit/5ba868f02493a9d81637d7f5983a47536332d7c8